### ISSUE TITLE
Add optional splits for items.

### ISSUE THREAD

**Issue by xThuby** [OliverAThunes/AM2RAutosplitter#4]
Splits for:
First metroid
nth item (missiles, e-tank etc).

The idea is to have better support for other categories like 100%.

**Comment by Miepee** [comment]
Adding to this, is it possible to have Autosplitter presets?
So if a runner switches frequently between any% and hundo, that they can just select the presets instead of having to deal with all of the optional splits.

**Comment by DodoBirby** [comment]
First metroid is easy enough. What numbers should the items be? Like 1st, half and last?

**Comment by DodoBirby** [comment]
Though specific locations are also possible since I have found the Global.item array.

**Comment by OliverAThunes** [comment]
When I wrote nth item I was thinking we have different settings for it, I.E every 4th item, every 6th item or 10th and so on. But perhaps splitting on specific items would be better.

**Comment by DodoBirby** [comment]
Yeah, I'll work on getting naming all the item location memory addresses since there are a lot. This'll probably be the first subcategory we add.
